packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -205,11 +224,20 @@ export const ChatList = ({
     onClick: () => markAllThreadsAsRead(chatListState?.chatThreads)
   };
 
+  const isLoading = [
```

Comment:
maybe we should have a isLoading state?

Reply:
`chatListState.status` already manages states. This is consistent with what the PG did. I don't know why we would add more state.

Reply:
because we can have more isLoading states in the future, right now, it's a string. This means if there are more isLoading types of states, this code does not need to change.

Reply:
The quality of "isLoading" is derivative of the status. Creating an additional state property and managing that adding code adds complexity. Of course any new states might need to be added to isLoading or isError, but that's pretty clear.

Anyway, that's the way the PG did theirs and I agree with the decision, so unless there is a compelling reason to change, I think it can stay.

Reply:
ok, I think we differ on this point (because I still feel it's not clear) - I know we also had reworked a lot of PG code, so it's not a reason to say it's the way PG did theirs a particular way and we are following. Regardless, I will not hold this PR.